package/gluon-mesh-vpn-wireguard/files/usr/bin/wg-broker

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+run_broker() {
+  local interface="$1"
+  local pubkey="$2"
+  local port
+  local interval=5
+
+  remote=$(wg show $interface endpoints|cut -d: -f1|awk '{print $2}')
+  group_peers=$(uci show wireguard|grep mesh_vpn_backbone_peer|grep =peer|cut -d"." -f2|cut -d= -f1)
+
+  for i in $group_peers
+  do
+    if [ $interface == $(uci get wireguard.$i.ifname) ]; then
+      thispeer=$i
+      break;
+    fi
+  done
+  remote=$(uci get wireguard.$thispeer.remote)
+
+  localport=$(wg show $interface listen-port)
+
+  while [[ "x$port" == "x" ]]
+  do
+    serverreply="$(echo "{\"pubkey\":\"$pubkey\", \"localport\":$localport }" | /usr/bin/netcat $remote 40000)"
```

Comment:
Hardcoded port

Reply:
Variable remote use `uci get wireguard.$thispeer.remote` it should be `uci get wireguard.$thispeer.broker`